cpp/include/cudf/io/parquet.hpp

```diff
@@ -394,8 +435,12 @@ class parquet_writer_options {
   statistics_freq _stats_level = statistics_freq::STATISTICS_ROWGROUP;
   // Sets of columns to output
   table_view _table;
+  // TODO: Remove. And maybe rename the new input_metadata/schema thing with just "metadata"
   // Optional associated metadata
   const table_metadata* _metadata = nullptr;
+
+  // TODO: Docs
+  table_input_metadata const* _table_meta = nullptr;
   // Parquet writes can write INT96 or TIMESTAMP_MICROS. Defaults to TIMESTAMP_MICROS.
   bool _write_timestamps_as_int96 = false;
```

Comment:
The one in column_in_metadata controls this on a per column level. I think @vuule wanted an option to still be able to control this with a single switch for the entire writer. This is not a pretty solution. I wanted to do this with a more content aware `table_input_metadata` class, that could handle setting an option for entire hierarchies underneath. But I didn't want to do this in this PR.

Reply:
I'd like to avoid breaking the API if we can, being able to apply this option to all columns might be useful to some users. If keeping this option is burdensome, we can check its usefulness with the Spark team.